packages/ui-components/src/components/AppShell/AppShell.component.tsx

```diff
@@ -36,12 +37,28 @@ export const AppShell = ({
     )
   }
 
+  const renderHeaderContainer = (
+    pageHeader?: AppShellProps["pageHeader"],
+    topNavigation?: AppShellProps["topNavigation"]
+  ) => {
+    if (!pageHeader && !topNavigation) {
+      return null
+    }
+    return (
+      <HeaderContainer fullWidth={fullWidthContent === true}>
+        {pageHeader && typeof pageHeader === "string" ? <PageHeader heading={pageHeader} /> : pageHeader}
```

Comment:
**(Non-Blocking Improvement)**

We already check `if (!pageHeader && !topNavigation)`

```suggestion
        {typeof pageHeader === "string" ? <PageHeader heading={pageHeader} /> : pageHeader}
```

Reply:
Wait, nvm, oopsy. We only check if both `true`. 🤯